src/docdb/tree/DocDBDocumentsTreeItem.ts

```diff
@@ -61,9 +61,20 @@ export class DocDBDocumentsTreeItem extends DocDBTreeItemBase<RetrievedDocument>
 
         if (docID || docID === "") {
             docID = docID.trim();
+            let body = { 'id': docID };
+            const partitionKey: string | undefined = this._collection.partitionKey && this._collection.partitionKey.paths[0];
```

Comment:
paths[0] is checking that the first array value has a value, but will throw if the array is empty.  Did you mean && this._collection.partitionKey.paths.length
?